builder/src/build.ts

```diff
@@ -145,8 +145,14 @@ export namespace Build {
 
       // Handle styles.
       // We explicitly ignore themes so they can be loaded dynamically.
-      if (data.style && !data.jupyterlab.themePath) {
-        cssImports.push(`${name}/${data.style}`);
+      if (!data.jupyterlab.themePath) {
+        // We prefer the styleModule key if it exists, falling back to
+        // the normal style key.
```

Comment:
Should `styleModule` be mentioned in the extension developer docs? I guess this could be addressed in #9252. 

If so maybe we'll also want to add it to the cookiecutter later on.

Reply:
Yes. I was going to address it in #9252.

Reply:
There is already a TODO item listed there for it.

Reply:
Perfect, thanks :+1: